C_routing.md

```diff
@@ -816,7 +797,8 @@ defmodule HelloPhoenix.Router do
   ...
 end
 ```
-Next, we need to use the `channel/3` macro to match a topic to the channel which will handle its requests. If we have a channel module called `RoomChannel` and a topic called `"rooms:*"`, the code to do this is straightforward.
+
```

Comment:
I think it's very important to keep the original here for two reasons.
- We need to call out that `channel/3` is a macro, just like `get/3` or `resources/3` above.
- Our topic is not 'lobby' . The topic is 'rooms:*'.